src/API/Recruit.Api/Queries/GetLiveVacanciesQueryHandler.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Esfa.Recruit.Vacancies.Client.Infrastructure.QueryStore;
+using MediatR;
+using SFA.DAS.Recruit.Api.Helpers;
+using SFA.DAS.Recruit.Api.Models;
+
+namespace SFA.DAS.Recruit.Api.Queries;
+
+public class GetLiveVacanciesQueryHandler : IRequestHandler<GetLiveVacanciesQuery, GetLiveVacanciesQueryResponse>
+{
+    private readonly IQueryStoreReader _queryStoreReader;
+
+    public GetLiveVacanciesQueryHandler(IQueryStoreReader queryStoreReader)
+    {
+        _queryStoreReader = queryStoreReader;
+    }
+
+    public async Task<GetLiveVacanciesQueryResponse> Handle(GetLiveVacanciesQuery request, CancellationToken cancellationToken)
+    {
+        var liveVacanciesToFetch = request.PageSize * request.PageNumber;
```

Comment:
I think this should be the Vacancies to skip, as in Ive requested page 5, and my page size is 1000, then i need to skip the first 5000 records and then get from 5001-6000 for my next page